benchmarks/pcvl_data.py

```diff
@@ -0,0 +1,92 @@
+import quandelibc as qc
```

Comment:
Don't you think you need to leave the license as in their GitHub file ?

Reply:
Don't know what you are talking about

Reply:
Up to line 23 https://github.com/Quandela/Perceval/blob/main/scripts/performance.py